### PR TITLE
Arrow functions

### PR DESCRIPTION
**Pull request by NickWilsonDev**
Hey @ponchieponcho please review

### CODE REVIEW

File: simpleServer.js [hunk 1]
```diff
@@ -48,14 +48,14 @@ let postContacts = function(request, response, id) {
     });
 };
 
-let updateContact = function(request, response, id) {
+let updateContact = (request, response, id) => {
     //update contact
```
Comment (ponchieponcho) [inline simpleServer.js:52]: Either add more description or remove

File: simpleServer.js [hunk 2]
```diff
@@ -33,13 +33,13 @@ let getContacts = function(request, response, id) {
...
     let body = '';
-    request.on('data', function(chunk) {
+    request.on('data', (chunk) => {
         body += chunk.toString();
     });
-    request.on('end', function() {
+    request.on('end', () => {
         let contact = JSON.parse(body);
         contact.id = ++lastId;
         console.log('Created: ' + contact);
```
Comment (ponchieponcho) [inline simpleServer.js:45]: If I understand this right, this code is just adding the new contact but isn't pushing back to the JSON file so it never changes. If not, I would add the functionality here.

**Comment by ponchieponcho** [comment]
Looks good...makes me think I should split out my phonebook code from the server. Merge 👍

**Comment by dbaile6** [comment]
Love the getPika variable.

**Pull request merged**